src/algebra/lie/of_associative.lean

```diff
@@ -62,6 +62,8 @@ instance of_associative_ring : lie_ring A :=
 
 lemma of_associative_ring_bracket (x y : A) : ⁅x, y⁆ = x*y - y*x := rfl
 
+lemma lie_apply {α : Type*} (f g : α → A) (a : α) : ⁅f, g⁆ a = ⁅f a, g a⁆ := rfl
```

Comment:
```suggestion
@[simp] lemma lie_apply {α : Type*} (f g : α → A) (a : α) : ⁅f, g⁆ a = ⁅f a, g a⁆ := rfl
```

Reply:
I actually had a reason not to have this simp, like some other `simp` lemma that was prevented by this, but since I don't remember it I will take the suggestion!

Reply:
If it turns out that there is a conflict, then it would be good to document it in a comment above this lemma.